service/history/historyEngine.go

```diff
@@ -1931,12 +1933,27 @@ func (e *historyEngineImpl) SignalWithStartWorkflowExecution(
 	// Start workflow and signal
 	startRequest := e.getStartRequest(namespaceID, sRequest)
 	request := startRequest.StartRequest
-	e.overrideStartWorkflowExecutionRequest(namespaceEntry, request, metrics.HistorySignalWorkflowExecutionScope)
-	err = e.validateStartWorkflowExecutionRequest(request, e.config.MaxIDLengthLimit(), namespaceEntry.GetInfo().GetName())
+	// QUESTION: should this be metrics.HistorySignalWithStartWorkflowExecutionScope?
+	e.overrideStartWorkflowExecutionRequest(namespace, request, metrics.HistorySignalWorkflowExecutionScope)
+	err = e.validateStartWorkflowExecutionRequest(request, namespace, metrics.HistorySignalWithStartWorkflowExecutionScope)
 	if err != nil {
 		return nil, err
 	}
 
+	if err := common.CheckEventBlobSizeLimit(
+		sRequest.GetSignalInput().Size(),
+		e.config.BlobSizeLimitWarn(namespace),
+		e.config.BlobSizeLimitError(namespace),
+		namespaceID,
+		sRequest.GetWorkflowId(),
+		"",
+		e.metricsClient.Scope(metrics.HistorySignalWithStartWorkflowExecutionScope),
```

Comment:
try to do the same as frontend
`wh.metricsScope(ctx).Tagged(metrics.CommandTypeTag(enumspb.COMMAND_TYPE_UNSPECIFIED.String())),`